core/src/apps/bitcoin/sign_tx/bitcoin.py

```diff
@@ -114,7 +127,12 @@ async def step1_process_inputs(self) -> None:
             self.weight.add_input(txi)
             if input_is_segwit(txi):
```

Comment:
If `txi` is an external segwit input but the witness is not send in this step (but it is in the step four), the transaction won't be serialised properly (in the step 7). As far as I know this is not security issue. Am I right?

Reply:
Yes.